integral_view/views/system_date_time.py

```diff
@@ -0,0 +1,44 @@
+import django
+import django.template
+
+from integral_view.forms import date_time_form
+from integralstor.update_system_date_time import update_system_date_time
```

Comment:
The date time utility functions should go into integralstor_utils. Also, pls call that fill date_time.py and not update_system_date_time.py